src/api/api.ts

```diff
@@ -0,0 +1,27 @@
+import axios, { Method } from 'axios';
+
+// Base Axios instance
+const api = axios.create({
+    baseURL: 'http://localhost:8080/',
```

Comment:
Can you check if the axios is working with the new URL Seb set up? Once it is working with that URL, I will approve it.